### DISCUSSION TITLE
Saving any tabular data displayed in various search features

### DISCUSSION THREAD

**Discussion by Dboingue** [winsiderss/systeminformer#2031] [Questions / Answers]
I seem to recall that processhacker, had many of it tabular output displays windows allowing for easy saving, and some filtering.
I would not have tried all features, but it did.  I tried now, and many of the windows with such output, have not much context menu other that copy, the keyboard shortcut crtl-a  does nothing, and one has to mouse select while scrolling to get all the information.

More over, it seemed, that the copy of that from context menu, has to UTF16 encoding, as I tried to past in a libreoffice calc document, and that was the only encoding that would not bring funny symbols instead of what I had seen with the systeminformer UI.

It could be LO cacl interpreting the text dump.  So this is 2 questions in one, but related to saving the queried information in general.

**Comment by Advait251206** [comment]
From what I've seen, System Informer doesn't provide a consistent export/save experience across all of its search and inspection windows. Some views support richer list operations, while others only expose basic actions such as **Copy** through the context menu.

A few observations:

- `Ctrl+A` is not universally implemented in every results window.
- Some dialogs use custom controls that don't support selecting all rows or exporting directly.
- In several cases, the only built-in option is copying the visible data to the clipboard.

Regarding the encoding issue, System Informer is a Unicode application and typically places text on the clipboard as UTF-16 (`CF_UNICODETEXT`). That's the standard format expected by most Windows applications.

If pasting into LibreOffice Calc produces unexpected characters, a few possibilities are:

- LibreOffice is interpreting the clipboard contents with the wrong encoding.
- The copied data contains tab/newline delimiters that Calc isn't parsing as expected.
- The source view is exporting data in a format that isn't ideal for spreadsheet import.

As a quick test, try pasting into:

- Notepad
- Notepad++
- VS Code

If the text appears correctly there, the issue is likely on the import/parsing side rather than with the clipboard data itself.

I agree that a more consistent experience would be useful. Features such as:

- `Ctrl+A` support in all tabular views.
- Copy selected rows vs. copy all rows.
- Export to CSV/TSV.
- Save results to a file.
- Built-in filtering/search within result dialogs.

would make it much easier to work with large datasets returned by the various search features.

Out of curiosity, which specific search/result window were you using (Handles, Strings, Memory Search, Network, etc.)? Some of them have slightly different implementations and capabilities.
